datasets/boreas.yaml

```diff
@@ -1,9 +1,9 @@
 Name: Boreas Autonomous Driving Dataset
-Description: This autonomous driving dataset includes data from a 128-beam Velodyne Alpha-Prime lidar, a 5MP Blackfly camera, a 360-degree Navtech radar, and post-processed Applanix POS LV GNSS data. This dataset was collect in various weather conditions (sun, rain, snow) over the course of a year. The intended purpose of this dataset is to enable benchmarking of long-term all-weather odometry and metric localization across various sensor types. In the future, we hope to also support an object detection benchmark.
+Description: "This autonomous driving dataset includes data from a 128-beam Velodyne Alpha-Prime lidar, a 5MP Blackfly camera, a 360-degree Navtech radar, and post-processed Applanix POS LV GNSS data. This dataset was collect in various weather conditions (sun, rain, snow) over the course of a year. The intended purpose of this dataset is to enable benchmarking of long-term all-weather odometry and metric localization across various sensor types. In the future, we hope to also support an object detection benchmark."
 Documentation: https://github.com/utiasASRL/pyboreas/blob/master/DATA_REFERENCE.md
 Contact: boreas@robotics.utias.utoronto.ca
 ManagedBy: "[ASRL](asrl.utias.utoronto.ca)"
-UpdateFrequency: New driving sequences will be added as they are collected.
+UpdateFrequency: "New driving sequences will be added as they are collected."
 Tags:
```

Comment:
included tags are invalid. See https://github.com/awslabs/open-data-registry/blob/main/tags.yaml for supported tag reference. You might check out https://github.com/awslabs/open-data-registry/blob/main/datasets/ford-multi-av-seasonal.yaml or https://github.com/awslabs/open-data-registry/blob/main/datasets/aev-a2d2.yaml as examples for recommended tags to autonomous driving datasets.